### ISSUE TITLE
Cluster deployed in non 'cnpg-system' namespace cannot find ConfigMap 'default-monitoring'

### ISSUE THREAD

**Issue by jacobfederer** [cloudnative-pg/cloudnative-pg#811]
Dear CNPG-Team,

We're currently evaluating CNPG for the use in our production environment. So far I'm quite amazed by the feature set. 

What I noticed was that if I deploy a cluster outside the cnpg-system namespace the monitoring does seem to work. The pods (for example in the default namespace) cannot access the config-map 'default-monitoring' with the monitoring config. The logs show the following: 

`{"level":"info","ts":1664860525.3757327,"msg":"Unable to get configMap containing custom monitoring queries","controller":"cluster","controllerGroup":"postgresql.cnpg.io","controllerKind":"Cluster","cluster":{"name":"cluster-example","namespace":"default"},"namespace":"default","name":"cluster-example","reconcileID":"20c93f14-17b6-4272-8ae2-75e10c317046","uuid":"8da77667-43a3-11ed-8574-aa174ec1f2ff","logging_pod":"cluster-example-1","reference":{"name":"cnpg-default-monitoring","key":"queries"},"error":"configmaps \"cnpg-default-monitoring\" not found"}`

### Environment:

CNPG-Sanbox: v0.5.1 
K8s-Variant: k3s v1.23.8 
System: Macos 12.6 on an M1 Mac

How should I go about this? My ideas were the following:
1. Somehow point the cluster monitoring config to the other namespace. 
2. Copy the default config map to the required namespace (Feels a bit hacky and seems like a lot of duplication)

Kind regards,
Jacob

**Comment by jacobfederer** [comment]
I read through the docs again and found that the config map is supposed to be copied automatically to the cluster namespace:
 https://github.com/cloudnative-pg/cloudnative-pg/blob/6d15ad78ba1836fd5dbd94751d6c850e9dcc52ae/docs/src/monitoring.md?plain=1#L548

So my issue might occur due to a misconfiguration of the default sandbox configuration. The sandbox points its config to default-monitoring-queries instead of cnpg-default-monitoring. 

I adjusted the config and after a reset/upgrade to 0.6.1 it seems to work. This is the config I used: 
cloudnative-pg:
  enabled: true
  config:
    create: true
    data:
      MONITORING_QUERIES_CONFIGMAP: cnpg-default-monitoring

**Comment by sxd** [comment]
@jacobfederer @phisco can you confirm that this was fixed?

**Comment by phisco** [comment]
@sxd yes, this was addressed here: https://github.com/cloudnative-pg/charts/pull/52
